ci/pipeline.yml

```diff
@@ -668,9 +668,14 @@ resources:
     server_side_encryption: AES256
 
 - name: cron-release
-  type: bosh-io-release
+  # type: bosh-io-release
+  # source:
+  #   repository: cloudfoundry-community/cron-boshrelease
+  # Use our own version until community release is updated
+  type: s3-iam
   source:
-    repository: cloudfoundry-community/cron-boshrelease
+    regexp: cron-(.*).tgz
+    <<: *s3-release-params
```

Comment:
While we're doing this, we need to make sure that the version is greater than the `deploy-bosh/cron-release` resource in order for our correct version to take hold. We've achieved this recently by copying over the `cron-1.1.4.tgz` release over to `cron-18.tgz` release because we had a cached `cron-17.tgz` release in there under `s3-iam`. cc: @sharms @jontours